### ISSUE TITLE
[angle] Build error

### ISSUE THREAD

**Issue by AlekseyMuraviev** [microsoft/vcpkg#26738]
Package: angle[core]:x64-windows -> chromium_4472#3
**Host Environment**
- Host: x64-windows
- Compiler: MSVC 19.32.31332.0
-    vcpkg-tool version: 2022-09-01-dfb82802c8cc562ce3b665a904a65b22314de724
    vcpkg-scripts version: 1ddafcb6f 2022-09-08 (17 hours ago)

**To Reproduce**
`vcpkg install angle:x64-windows`

**Failure logs**
```
-- Using cached google-angle-d15be77864e18f407c317be6f6bc06ee2b7d070a.tar.gz.
-- Cleaning sources at D:/libs/vcpkg/buildtrees/angle/src/ee2b7d070a-83a419ab43.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/libs/vcpkg/downloads/google-angle-d15be77864e18f407c317be6f6bc06ee2b7d070a.tar.gz
-- Applying patch 001-fix-uwp.patch
-- Applying patch 002-fix-builder-error.patch
-- Applying patch 003-fix-mingw.patch
-- Using source at D:/libs/vcpkg/buildtrees/angle/src/ee2b7d070a-83a419ab43.clean
-- Using cached D:/libs/vcpkg/downloads/angle-09490503d0f201b81e03f5ca0ab8ba8ee76d4a8e.tar.gz
-- Extracting source D:/libs/vcpkg/downloads/angle-09490503d0f201b81e03f5ca0ab8ba8ee76d4a8e.tar.gz
-- Applying patch third-party-zlib-far-undef.patch
-- Using source at D:/libs/vcpkg/buildtrees/angle/src/8ee76d4a8e-895bfc7289.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:129 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" --build . --config Debug --target install -- -v -j17
    Working Directory: D:/libs/vcpkg/buildtrees/angle/x64-windows-dbg
    See logs for more information:
      D:\libs\vcpkg\buildtrees\angle\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:78 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:18 (vcpkg_build_cmake)
  ports/angle/portfile.cmake:64 (vcpkg_install_cmake)
  scripts/ports.cmake:147 (include)



```
<details><summary>D:\libs\vcpkg\buildtrees\angle\install-x64-windows-dbg-out.log</summary>

```
[1/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_STANDALONE_BUILD -DGL_SILENCE_DEPRECATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\angle_translator.dir\src\compiler\translator\blocklayoutHLSL.cpp.obj /FdCMakeFiles\angle_translator.dir\ /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\compiler\translator\blocklayoutHLSL.cpp
[2/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_STANDALONE_BUILD -DGL_SILENCE_DEPRECATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\angle_translator.dir\src\compiler\translator\tree_ops\d3d\SeparateExpressionsReturningArrays.cpp.obj /FdCMakeFiles\angle_translator.dir\ /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\compiler\translator\tree_ops\d3d\SeparateExpressionsReturningArrays.cpp
[3/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_STANDALONE_BUILD -DGL_SILENCE_DEPRECATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\angle_translator.dir\src\compiler\translator\tree_ops\d3d\UnfoldShortCircuitToIf.cpp.obj /FdCMakeFiles\angle_translator.dir\ /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\compiler\translator\tree_ops\d3d\UnfoldShortCircuitToIf.cpp
...
Skipped 518 lines
...

[354/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_CAPTURE_ENABLED=0 -DANGLE_ENABLE_D3D11 -DANGLE_ENABLE_D3D11_COMPOSITOR_NATIVE_WINDOW -DANGLE_ENABLE_D3D9 -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_ENABLE_NULL -DANGLE_ENABLE_OPENGL -DANGLE_ENABLE_OPENGL_DESKTOP -DANGLE_EXPORT="" -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES="{ \"d3dcompiler_47.dll\", \"d3dcompiler_46.dll\", \"d3dcompiler_43.dll\" }" -DANGLE_STANDALONE_BUILD -DANGLE_UTIL_EXPORT="" -DEGLAPI="" -DGL_API="" -DGL_APICALL="" -DGL_SILENCE_DEPRECATION -DLIBANGLE_IMPLEMENTATION -DLIBANGLE_UTIL_IMPLEMENTATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\third_party\khronos -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\third_party\zlib\google -external:ID:\libs\vcpkg\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\libANGLE.dir\src\libANGLE\renderer\d3d\d3d9\StateManager9.cpp.obj /FdCMakeFiles\libANGLE.dir\libANGLE.pdb /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\libANGLE\renderer\d3d\d3d9\StateManager9.cpp
[355/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_CAPTURE_ENABLED=0 -DANGLE_ENABLE_D3D11 -DANGLE_ENABLE_D3D11_COMPOSITOR_NATIVE_WINDOW -DANGLE_ENABLE_D3D9 -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_ENABLE_NULL -DANGLE_ENABLE_OPENGL -DANGLE_ENABLE_OPENGL_DESKTOP -DANGLE_EXPORT="" -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES="{ \"d3dcompiler_47.dll\", \"d3dcompiler_46.dll\", \"d3dcompiler_43.dll\" }" -DANGLE_STANDALONE_BUILD -DANGLE_UTIL_EXPORT="" -DEGLAPI="" -DGL_API="" -DGL_APICALL="" -DGL_SILENCE_DEPRECATION -DLIBANGLE_IMPLEMENTATION -DLIBANGLE_UTIL_IMPLEMENTATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\third_party\khronos -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\third_party\zlib\google -external:ID:\libs\vcpkg\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\libANGLE.dir\src\libANGLE\renderer\d3d\d3d9\SwapChain9.cpp.obj /FdCMakeFiles\libANGLE.dir\libANGLE.pdb /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\libANGLE\renderer\d3d\d3d9\SwapChain9.cpp
[356/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_CAPTURE_ENABLED=0 -DANGLE_ENABLE_D3D11 -DANGLE_ENABLE_D3D11_COMPOSITOR_NATIVE_WINDOW -DANGLE_ENABLE_D3D9 -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_ENABLE_NULL -DANGLE_ENABLE_OPENGL -DANGLE_ENABLE_OPENGL_DESKTOP -DANGLE_EXPORT="" -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES="{ \"d3dcompiler_47.dll\", \"d3dcompiler_46.dll\", \"d3dcompiler_43.dll\" }" -DANGLE_STANDALONE_BUILD -DANGLE_UTIL_EXPORT="" -DEGLAPI="" -DGL_API="" -DGL_APICALL="" -DGL_SILENCE_DEPRECATION -DLIBANGLE_IMPLEMENTATION -DLIBANGLE_UTIL_IMPLEMENTATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\third_party\khronos -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\third_party\zlib\google -external:ID:\libs\vcpkg\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\libANGLE.dir\src\libANGLE\renderer\d3d\d3d9\formatutils9.cpp.obj /FdCMakeFiles\libANGLE.dir\libANGLE.pdb /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\libANGLE\renderer\d3d\d3d9\formatutils9.cpp
[357/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_CAPTURE_ENABLED=0 -DANGLE_ENABLE_D3D11 -DANGLE_ENABLE_D3D11_COMPOSITOR_NATIVE_WINDOW -DANGLE_ENABLE_D3D9 -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_ENABLE_NULL -DANGLE_ENABLE_OPENGL -DANGLE_ENABLE_OPENGL_DESKTOP -DANGLE_EXPORT="" -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES="{ \"d3dcompiler_47.dll\", \"d3dcompiler_46.dll\", \"d3dcompiler_43.dll\" }" -DANGLE_STANDALONE_BUILD -DANGLE_UTIL_EXPORT="" -DEGLAPI="" -DGL_API="" -DGL_APICALL="" -DGL_SILENCE_DEPRECATION -DLIBANGLE_IMPLEMENTATION -DLIBANGLE_UTIL_IMPLEMENTATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\third_party\khronos -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\third_party\zlib\google -external:ID:\libs\vcpkg\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\libANGLE.dir\src\libANGLE\renderer\d3d\d3d9\VertexBuffer9.cpp.obj /FdCMakeFiles\libANGLE.dir\libANGLE.pdb /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\libANGLE\renderer\d3d\d3d9\VertexBuffer9.cpp
[358/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_CAPTURE_ENABLED=0 -DANGLE_ENABLE_D3D11 -DANGLE_ENABLE_D3D11_COMPOSITOR_NATIVE_WINDOW -DANGLE_ENABLE_D3D9 -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_ENABLE_NULL -DANGLE_ENABLE_OPENGL -DANGLE_ENABLE_OPENGL_DESKTOP -DANGLE_EXPORT="" -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES="{ \"d3dcompiler_47.dll\", \"d3dcompiler_46.dll\", \"d3dcompiler_43.dll\" }" -DANGLE_STANDALONE_BUILD -DANGLE_UTIL_EXPORT="" -DEGLAPI="" -DGL_API="" -DGL_APICALL="" -DGL_SILENCE_DEPRECATION -DLIBANGLE_IMPLEMENTATION -DLIBANGLE_UTIL_IMPLEMENTATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\third_party\khronos -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\third_party\zlib\google -external:ID:\libs\vcpkg\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\libANGLE.dir\src\libANGLE\renderer\d3d\d3d9\TextureStorage9.cpp.obj /FdCMakeFiles\libANGLE.dir\libANGLE.pdb /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\libANGLE\renderer\d3d\d3d9\TextureStorage9.cpp
[359/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_CAPTURE_ENABLED=0 -DANGLE_ENABLE_D3D11 -DANGLE_ENABLE_D3D11_COMPOSITOR_NATIVE_WINDOW -DANGLE_ENABLE_D3D9 -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_ENABLE_NULL -DANGLE_ENABLE_OPENGL -DANGLE_ENABLE_OPENGL_DESKTOP -DANGLE_EXPORT="" -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES="{ \"d3dcompiler_47.dll\", \"d3dcompiler_46.dll\", \"d3dcompiler_43.dll\" }" -DANGLE_STANDALONE_BUILD -DANGLE_UTIL_EXPORT="" -DEGLAPI="" -DGL_API="" -DGL_APICALL="" -DGL_SILENCE_DEPRECATION -DLIBANGLE_IMPLEMENTATION -DLIBANGLE_UTIL_IMPLEMENTATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\third_party\khronos -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\third_party\zlib\google -external:ID:\libs\vcpkg\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\libANGLE.dir\src\libANGLE\renderer\d3d\d3d9\VertexDeclarationCache.cpp.obj /FdCMakeFiles\libANGLE.dir\libANGLE.pdb /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\libANGLE\renderer\d3d\d3d9\VertexDeclarationCache.cpp
[360/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_CAPTURE_ENABLED=0 -DANGLE_ENABLE_D3D11 -DANGLE_ENABLE_D3D11_COMPOSITOR_NATIVE_WINDOW -DANGLE_ENABLE_D3D9 -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_ENABLE_NULL -DANGLE_ENABLE_OPENGL -DANGLE_ENABLE_OPENGL_DESKTOP -DANGLE_EXPORT="" -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES="{ \"d3dcompiler_47.dll\", \"d3dcompiler_46.dll\", \"d3dcompiler_43.dll\" }" -DANGLE_STANDALONE_BUILD -DANGLE_UTIL_EXPORT="" -DEGLAPI="" -DGL_API="" -DGL_APICALL="" -DGL_SILENCE_DEPRECATION -DLIBANGLE_IMPLEMENTATION -DLIBANGLE_UTIL_IMPLEMENTATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\third_party\khronos -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\third_party\zlib\google -external:ID:\libs\vcpkg\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\libANGLE.dir\src\libANGLE\renderer\gl\ContextGL.cpp.obj /FdCMakeFiles\libANGLE.dir\libANGLE.pdb /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\libANGLE\renderer\gl\ContextGL.cpp
[361/419] C:\PROGRA~1\MICROS~4\2022\PROFES~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x64\cl.exe   /TP -DANGLE_CAPTURE_ENABLED=0 -DANGLE_ENABLE_D3D11 -DANGLE_ENABLE_D3D11_COMPOSITOR_NATIVE_WINDOW -DANGLE_ENABLE_D3D9 -DANGLE_ENABLE_ESSL -DANGLE_ENABLE_GLSL -DANGLE_ENABLE_HLSL -DANGLE_ENABLE_NULL -DANGLE_ENABLE_OPENGL -DANGLE_ENABLE_OPENGL_DESKTOP -DANGLE_EXPORT="" -DANGLE_IS_64_BIT_CPU -DANGLE_IS_WIN -DANGLE_PRELOADED_D3DCOMPILER_MODULE_NAMES="{ \"d3dcompiler_47.dll\", \"d3dcompiler_46.dll\", \"d3dcompiler_43.dll\" }" -DANGLE_STANDALONE_BUILD -DANGLE_UTIL_EXPORT="" -DEGLAPI="" -DGL_API="" -DGL_APICALL="" -DGL_SILENCE_DEPRECATION -DLIBANGLE_IMPLEMENTATION -DLIBANGLE_UTIL_IMPLEMENTATION -DNOMINMAX -D_CRT_SECURE_NO_DEPRECATE -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_WARNINGS -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src -ID:\libs\vcpkg\buildtrees\angle\x64-windows-dbg\include -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\third_party\khronos -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\common\third_party\base -ID:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\third_party\zlib\google -external:ID:\libs\vcpkg\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /d2guard4 /Wv:18 /guard:cf /permissive /bigobj -DANGLE_ENABLE_DEBUG_ANNOTATIONS /showIncludes /FoCMakeFiles\libANGLE.dir\src\libANGLE\renderer\gl\BufferGL.cpp.obj /FdCMakeFiles\libANGLE.dir\libANGLE.pdb /FS -c D:\libs\vcpkg\buildtrees\angle\src\ee2b7d070a-83a419ab43.clean\src\libANGLE\renderer\gl\BufferGL.cpp
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

**Comment by Cheney-W** [comment]
Could you please send the `install-x64-windows-dbg-out.log` file to me?

**Comment by Kolfering** [comment]
There was a known compiler bug when building angle with VS 17.2 (or msvc 19.32) which was fixed in the 17.3 version. See #24704 and try to update your VS version, its most likely the same bug you are encountering.

**Comment by Cheney-W** [comment]
Thanks for posting this issue. Please reopen this issue if this is still a problem for you.
